lib/font_awesome/less/version.rb

```diff
@@ -1,5 +1,5 @@
 module FontAwesome
   module Less
-    VERSION = '4.2.0'
+    VERSION = '4.4.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.